src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIfStatement.cs

```diff
@@ -1190,15 +1190,14 @@ void M()
     Predecessors (1)
         [0]
     Statements (1)
-        IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'bool condition = false;')
-          IVariableDeclarationOperation (1 declarators) (OperationKind.VariableDeclaration, Type: null) (Syntax: 'bool condition = false')
-            Declarators:
-                IVariableDeclaratorOperation (Symbol: System.Boolean condition) (OperationKind.VariableDeclarator, Type: null) (Syntax: 'condition = false')
-                  Initializer: 
-                    IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null) (Syntax: '= false')
-                      ILiteralOperation (OperationKind.Literal, Type: System.Boolean, Constant: False) (Syntax: 'false')
-            Initializer: 
-              null
+        IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: 'condition = false')
```

Comment:
I recall us talking to rewrite variable initialization into simple assignment, but I am not sure why are we removing the IVariableDeclaration operations itself? I am not sure if this aids any further in control flow analysis. In fact, if some client wants to do some variable analysis and report a diagnostic on the variable declaration, removing the `IVariableDeclarationOperation` nodes makes it difficult for them.

Reply:
The IVariableDeclaration and the initializer aren't guaranteed to be in the same place, so clients will have to handle this anyway. If the client needs to report a diagnostic on the symbol's original definition, they'd have to go through `DeclaringSyntaxReferences` anyway.

Reply:
Yes, but I am not sure what is the value add or necessity of removing `IVariableDeclaration` operation for flow analysis?

Reply:
There's no benefit to leaving it in, and it adds unnecessary nodes to the graph. We've already said that we're going to have variable lifetime information in a separate data structure, and all the `IVariableDeclaration` does (or really `IVariableDeclarator`, which is what I think you mean here) is provide that information.

Reply:
@jinujoseph, we'll bring this up in the meeting today.

Reply:
Per design discussion today, we'll fully remove the IVariableDeclarat(ion(Group))(or) from the resulting graph.